connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorSourceConnector.java

```diff
@@ -83,6 +84,12 @@ public MirrorSourceConnector() {
         // nop
     }
 
+    // visible for testing
+    public MirrorSourceConnector(List<TopicPartition> knownTopicPartitions, MirrorConnectorConfig config) {
```

Comment:
You can drop the "public" here, since the tests are in the same package.

Reply:
👍  updated